### PR TITLE
update metadata for release `1.13.0`

### PR DESCRIPTION
**Pull request by zacharyburnett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ the specified context and less than the context for the next release.
 
 | jwst tag            | DMS build | SDP_VER  | CRDS_CONTEXT | Released   | Ops Install | Notes                                         |
 |---------------------|-----------|----------|--------------|------------|-------------|-----------------------------------------------|
+| 1.13.0              | B10.1rc1  | 2023.3.1 | 1176         | 2023-12-15 |             | First release candidate for B10.1             |
```
Comment (zacharyburnett) [inline README.md:213]: should the `SDP_VER` be `TBD` for now?

Comment (hbushouse) [reply README.md:213]: Let's start with it at 2023.4.0 for now.

File: pyproject.toml [hunk 2]
```diff
@@ -36,7 +36,7 @@ dependencies = [
     "scikit-image>=0.19",
     "scipy>=1.6.0",
     "spherical-geometry>=1.2.22",
-    "stcal>=1.4.4,<1.5.0",
+    "stcal>=1.4.4,<1.6.0",
```
Comment (hbushouse) [inline pyproject.toml:39]: The latest stcal is 1.5.2, so shouldn't this have a higher minimum value?

**Review by hbushouse — APPROVED** [review]
LGTM

**Pull request merged**